libs/entry-components/header/styles/modules/components/headers/_general.scss

```diff
@@ -7,6 +7,7 @@
 
         .title {
             font-size: map.value-from($theme, 'header.title-size');
+            color: map.value-from($theme, 'header.title-color');
```

Comment:
You changed the structure of object. This won't work anymore => 'header.title-color'
You also need to change it to =>'header.title.color' in order to work. Same for size.